### PR TITLE
Add pre-commit hook

### LINKED ISSUE #5: JS formatting pre-commit hook
Ensure that JS code is formatted via `prettier` before commit

A possible option: https://prettier.io/docs/en/precommit.html#option-2-pretty-quickhttpsgithubcomazzpretty-quick

### PR DESCRIPTION
**Pull request by smorimoto**
Closes: #5

### CODE REVIEW

**Comment by smorimoto** [comment]
This PR is ready for review.

**Comment by walfie** [comment]
whoops, merging the other PR caused conflicts in this one

**Comment by smorimoto** [comment]
Just resolved the conflict

File: package.json [hunk 1]
```diff
@@ -6,32 +6,42 @@
   "scripts": {
     "predev": "yarn run minify-svg",
     "dev": "parcel src/index.html",
-    "fmt": "ocamlformat --inplace src/*.ml src/*.mli",
-    "//": [
-      "Build twice as a terrible hack because pwa-manifest runs *after* sw-cache,",
-      "so the latter doesn't know about the generated icon/webmanifest files..."
```
Comment (walfie) [inline package.json:12]: I wish there was a way to add real comments to package.json...

**Review by walfie — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (smorimoto) [reply package.json:12]: I thought prettier couldn't format it, but the editor warned me about a different problem.

**Pull request merged**